packages/client/src/Components/MessageCreator/MessageCreator.tsx

```diff
@@ -83,6 +85,41 @@ const MessageCreator: React.FC<Props> = (props) => {
   }, [props])
 
   const createEditor = (schema: any) => {
+    /*
+    ** Render Default datetime entries in template of json for type datetime-local
```

Comment:
@sanjayradadiya - this looks like it loops through the top level properties in the schema.

But, the date may be in a child object.  So, I think we actually need to "walk" the schema if the property is of `"type": "object"`:
![image](https://user-images.githubusercontent.com/1108513/126161729-f6b8ec71-a3cd-45d9-8048-734098be71c9.png)